docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,26 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Mirroring and rewriting the docker.io registry
+
+The default Docker registry `docker.io` is handled in a special way, because
+every push and pull operation gets internally suffixed with `/library`. This
+means that a pull of `docker.io/image` will be internally translated to
+`docker.io/library/image`. This has to be considered when rewriting registries
+under that domain.
```

Comment:
I think it's worth highlighting two more things:
* We need this to match Docker's behavior. Some folks might be wondering why there's such special treatment.
* It's only rewritten, if the specified reference is without a repository. For instance, `docker.io/opensuse/leap` will not be rewritten. I think that's worth mentioning to avoid misinterpreting the example.

Reply:
Alright, thanks for the detailed review. I'll adapted the examples to match a real world use case and explicitly mentioned that it only affects the unqualified docker paths.